index.html

```diff
@@ -20,12 +20,17 @@
             <h1>
                 <picture>
                     <source media="(max-width:854px)" srcset="photos\mobile_rose.png" />
-                    <source media="(min-width:854px)" srcset="photos\desktop_rose.png" />
+                    <source media="(min-width:854px)" width="395px" height="38px" srcset="photos\desktop_rose.png" />
                     <img class="rose" width="217px" height="24px" src="photos\mobile_rose.png" alt="NORDIC ROSE">
                 </picture>
             </h1>
         </div>
-
+        <div class="desktop_menu">
```

Comment:
if you're interested in how to organize your class names better, you can read about BEM methodology [here](https://getbem.com/introduction/). There are a lot of other methodologies, this one is a good to start with. Andrey is also using it